petrinet/examples/sir.json

```diff
@@ -68,7 +76,7 @@
           "description": "infection rate",
           "value": 0.027,
           "distribution": {
-            "type": "StndardUniform1",
+            "type": "StandardUniform1",
```

Comment:
I think since we use ProbOnto by convention and code names are unique we don't have to at this point. The idea of generating a JSON schema from ProbOnto to be able to validate these distribution objects has come up.